lib/components/Editor/Menu/Fixed/LinkOption.jsx

```diff
@@ -44,23 +40,17 @@ const LinkOption = ({ editor }) => {
 
   return (
     <Dropdown
-      className="neeto-editor-link-wrapper"
+      buttonStyle={isActive ? "secondary" : "text"}
       closeOnSelect={false}
+      data-cy="neeto-editor-fixed-menu-link-option"
+      icon={() => <Link size={18} />}
       isOpen={isOpen}
       position="bottom"
-      customTarget={() => (
-        <MenuButton
-          data-cy="neeto-editor-fixed-menu-link-option"
-          icon={Link}
-          iconActive={isActive}
-          tooltipProps={{ content: "Link", position: "bottom", delay: [500] }}
-          onClick={onClickTrigger}
-        />
-      )}
+      tooltipProps={{ content: "Link", position: "bottom" }}
```

Comment:
Here we need to pass `tooltipProps` inside `buttonProps` since it is a dropdown.